src/python/pants/backend/helm/target_types.py

```diff
@@ -385,6 +388,39 @@ def all_helm_artifact_targets(all_targets: AllTargets) -> AllHelmArtifactTargets
 # -----------------------------------------------------------------------------------------------
 
 
+class HelmDeploymentChartField(StringField, AsyncFieldMixin):
```

Comment:
Can you talk about why we don't use SpecialCasedDependencies here? Is it because we might be depending on a Helm chart from a repository, which wouldn't be a Pants dependency? something like `helm_deployment(chart="bitnami/wordpress")`
I'm wondering about the reasoning so I can think it through for Terraform.

Reply:
Well, it's the same reason of moving away from using the `dependencies` field: both take a list of values while I would like to enforce that is one, and only one mandatory value.

Reply:
That makes sense, thanks. I think that'll be useful for Terraform as well